cupy/fft/_cache.pyx

```diff
@@ -263,6 +264,11 @@ cdef class PlanCache:
     # ---------------------- Python methods ---------------------- #
 
     def __init__(self, Py_ssize_t size=16, Py_ssize_t memsize=-1, int dev=-1):
+        if runtime.runtimeGetVersion() == 11010:
```

Comment:
We would like to do a more strict check here, but since this happens with prime sizes, add primality tests here can be overkill.

Reply:
That would incur extra overhead and actually requires some knowledge about cufft internal for a proper test, so it's easier to just kill the cache unconditionally.